### PR TITLE
New package: sharenix-0.10.0a

### PR DESCRIPTION
**Pull request by Francesco149**
this tool uploads files to various file and image sharing services and is somewhat compatible with windows ShareX json configs

tested on x86_64

### CODE REVIEW

**Comment by Francesco149** [comment]
oops, i guess git should be hostmakedepends? arm and aarch64 are failing to find git

**Comment by Francesco149** [comment]
yep, all builds passing now

File: srcpkgs/sharenix/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'sharenix'
+pkgname=sharenix
+version=0.10.0a
+revision=1
+build_style=go
+go_import_path="github.com/Francesco149/sharenix"
+hostmakedepends="pkg-config git"
+makedepends="gtk+-devel"
+depends="gtk+ gnome-screenshot"
```
Comment (maxice8) [inline srcpkgs/sharenix/template]: no need for gtk+ it is found via `shlibs`

File: srcpkgs/sharenix/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+short_desc="ShareX clone for linux and FreeBSD"
+maintainer="Franc[e]sco <lolisamurai@tfwno.gf>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/Francesco149/sharenix"
+distfiles="https://github.com/Francesco149/sharenix/archive/${version}.tar.gz"
+checksum=45f00008b3e717d8d11ffd97d91811539e335f618f878c65bc8e2c9080618178
+
+post_install() {
+	vlicense LICENSE
```
Comment (Duncaen) [inline srcpkgs/sharenix/template]: Don't need to install yet another copy of the GPL license, the standard ones are already in `/usr/share/licenses`.

**Comment by Francesco149** [comment]
thanks for the help, fixed both issues

**Comment by Francesco149** [comment]
removed hard dependency on gnome-screenshot as it's optional. instead, a post install message is displayed with the list of the compatible screenshotting tool (which are also optional)

**Comment by Duncaen** [comment]
I don't think we should start to list optional dependencies in `INSTALL.msg`.

**Comment by Francesco149** [comment]
@Duncaen where would be a good place?

**Comment by Johnnynator** [comment]
Creating a virtual package dep would be the correct (and more complicated) approach.

**Comment by Duncaen** [comment]
Probably documentation or error messages of the program because we don't really have optional dependencies.

**Pull request merged**